examples/README.md

```diff
@@ -42,6 +42,18 @@ Some notebooks may require additional data. This data will be downloaded when ru
 
 For issues relating to XAITK-Saliency functionality or running of an example, please create an issue on the [repository](https://github.com/XAITK/xaitk-saliency/issues).
 
+
+### Continuous Integration
+Some notebooks contained here are executed as part of the continuous
+integration workflows to ensure their example appropriately reflects the
+toolkit's current functionality.
+Not all are included in this CI due to their long-running or heavy resource
+requirements, e.g. the ``covid_classification.ipynb`` which trains a deep
+learning model which at the very least takes a non-trivial amount of time.
+
+The CI workflow may be found in ``.github/workflows/ci-example-notebooks.yml``.
+The specification of which notebooks are included in CI is around L29.
```

Comment:
The documentation here is great- do you think we need to update `docs/review_process.rst` in our ReadTheDocs to account for notebooks also being covered by CI? Or stated another way, what should a contributor who adds a demo notebook expect in terms of the review process?

Reply:
Valid point. I can add an info section to the review-process doc, including a note for the human reviewer to check for CI inclusion of new notebooks. It sounds like we should add to the ``CONTRIBUTING.md`` about adding notebooks to the CI upon that sort of contribution.